forge-gui/res/cardsfolder/upcoming/stangg_echo_warrior.txt

```diff
@@ -0,0 +1,15 @@
+Name:Stangg, Echo Warrior
+ManaCost:2 R G
+Types:Legendary Creature Human Warrior
+PT:3/4
+T:Mode$ Attacks | ValidCard$ Card.Self | TriggerZones$ Battlefield | Execute$ TrigStangg | TriggerDescription$ Whenever CARDNAME attacks, create Stangg Twin, a legendary 3/4 red and green Human Warrior creature token. It enters the battlefield tapped and attacking. For each Aura and Equipment attached to Stangg, create a token that's a copy of it attached to Stangg Twin. Sacrifice all tokens created this way at the beginning of the next end step.
+SVar:TrigStangg:DB$ Token | TokenScript$ stangg_twin | TokenOwner$ You | ImprintTokens$ True | TokenAttacking$ True | TokenTapped$ True  | SubAbility$ CreateCopy
+SVar:CreateCopy:DB$ CopyPermanent | Defined$ Valid Equipment.Attached,Aura.Attached | NumCopies$ 1 | AttachedTo$ Imprinted | RememberTokens$ True | SubAbility$ ExileTokens
+SVar:ExileTokens:DB$ DelayedTrigger | Mode$ Phase | Phase$ End of Turn | ValidPlayer$ Player | Execute$ SacAll | TriggerDescription$ Sacrifice all tokens created this way at the beginning of the next end step.
```

Comment:
The DelayedTrigger is probably not so good, because the Sacrifice below doesn't use DelayedTrigRemembered


as for the Sacrifice stuff, please try this, the Stangg Twin Token is also remembered, all of the remembered Tokens are getting Marked with AtEOT, and the Delayed Trigger is handled by the engine:

```
SVar:TrigStangg:DB$ Token | TokenScript$ stangg_twin | TokenOwner$ You | ImprintTokens$ True  | RememberTokens$ True | TokenAttacking$ True | TokenTapped$ True  | SubAbility$ CreateCopy
SVar:CreateCopy:DB$ CopyPermanent | Defined$ Valid Equipment.Attached,Aura.Attached | NumCopies$ 1 | AttachedTo$ Imprinted | RememberTokens$ True | SubAbility$ ExileTokens
SVar:ExileTokens:DB$ Pump | Defined$ Remembered | AtEOT$ Sacrifice | SubAbility$ DBCleanup
SVar:DBCleanup:DB$ Cleanup | ClearImprinted$ True | ClearRemembered$ True
```